### PR TITLE
Introduce duplicated field maxSizeMbStaticDataSources in Plan to rena…

### PR DESCRIPTION
**Pull request by PopDaph**
To deploy: 
```
./admin/init_db.sh
./admin/init_plans.sh
```

### CODE REVIEW

File: front/lib/models/plan.ts [hunk 1]
```diff
@@ -105,6 +106,11 @@ Plan.init(
       type: DataTypes.INTEGER,
       allowNull: false,
     },
+    maxSizeMbStaticDataSources: {
```
Comment (spolu) [inline front/lib/models/plan.ts:109]: No Static :) It is enforced for all data sources

**Review by spolu — COMMENTED** [review]
Looks good modulo Static in the name.

Let's also explicit a deploy procedure :pray:

File: front/lib/models/plan.ts (hunk 1 above)
Comment (PopDaph) [reply front/lib/models/plan.ts:109]: Yes today it is. I can rename to match the names for the current limitations PlanType, but we will have to rework and rename this again when we properly handle the limitations that we want in the pricing (the pricing discussion was still in progress when I implementing this). I don't think it's more urgent than working on plugging Stripe tho, but I'll do it 👍🏻

Comment (spolu) [reply front/lib/models/plan.ts:109]: Very well possible that this will be a "new" limitation also

Comment (PopDaph) [reply front/lib/models/plan.ts]: Yes of course, my point was only about priorities and pricing being more urgent 🙇🏻‍♀️ 
I added a commit to match exactly the names of the type, good for you? 🙏🏻

Comment (spolu) [reply front/lib/models/plan.ts]: yes yes!

**Review by spolu — APPROVED** [review]

**Pull request merged**